app/src/main/java/com/mediaan/mymediaan/model/Interests.kt

```diff
@@ -0,0 +1,28 @@
+package com.mediaan.mymediaan.model
+
+enum class Interest {
+    MUSIC,
+    SPORTS,
+    HIKING,
+    DOGS,
+    CATS,
```

Comment:
To make app a bit more inclusive, we can combine these two under name "animals". Then it also includes birds and other living things :D

Reply:
...I may or may not have asked ChatGPT to generate this list. Updating it.